README.md

```diff
@@ -63,33 +86,81 @@ module "eks-jx" {
+```sh
+export VAULT_AWS_ACCESS_KEY_ID=$(terraform output vault_user_id)
+export VAULT_AWS_SECRET_ACCESS_KEY=$(terraform output vault_user_secret)
+```
 
-#### EKS Worker Nodes configuration
-<a id="markdown-EKS%20Worker%20Nodes%20configuration" name="EKS%20Worker%20Nodes%20configuration"></a>
+If you specified _vault_user_ you need to provide the access key id and secret for the specified user.
 
-You can configure the EKS worker node pool with the following variables: `desired_number_of_nodes`, `min_number_of_nodes`, `max_number_of_nodes` and `worker_nodes_instance_types`.
+:warning: **Note**: This example is for getting up and running quickly.
+It is not intended for a production cluster.
+Refer to [Production cluster considerations](#production-cluster-considerations) for things to consider when creating a production cluster.
+
+The following sections provide a full list of configuration in- and output variables.
+
+#### Inputs
+<a id="markdown-Inputs" name="Inputs"></a>
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:-----:|
+| apex\_domain | The main domain to either use directly or to configure a subdomain from | `string` | `""` | no |
+| cluster\_name | Variable to provide your desired name for the cluster. The script will create a random name if this is empty | `string` | `""` | no |
+| create\_and\_configure\_subdomain | Flag to create an NS record ser for the subdomain in the apex domain's Hosted Zone | `bool` | `false` | no |
+| desired\_node\_count | The number of worker nodes to use for the cluster | `number` | `3` | no |
+| enable\_external\_dns | Flag to enable or disable External DNS in the final `jx-requirements.yml` file | `bool` | `false` | no |
+| enable\_logs\_storage | Flag to enable or disable long term storage for logs | `bool` | `true` | no |
+| enable\_reports\_storage | Flag to enable or disable long term storage for reports | `bool` | `true` | no |
+| enable\_repository\_storage | Flag to enable or disable the repository bucket storage | `bool` | `true` | no |
+| enable\_tls | Flag to enable TLS int he final `jx-requirements.yml` file | `bool` | `false` | no |
+| max\_node\_count | The maximum number of worker nodes to use for the cluster | `number` | `5` | no |
+| min\_node\_count | The minimum number of worker nodes to use for the cluster | `number` | `3` | no |
+| node\_machine\_type | The instance type to use for the cluster's worker nodes | `string` | `"m5.large"` | no |
+| production\_letsencrypt | Flag to use the production environment of letsencrypt in the `jx-requirements.yml` file | `bool` | `false` | no |
+| region | The region to create the resources into | `string` | `"us-east-1"` | no |
+| subdomain | The subdomain to be used added to the apex domain. If subdomain is set, it will be appended to the apex domain in  `jx-requirements-eks.yml` file | `string` | `""` | no |
```

Comment:
```suggestion
| subdomain | The subdomain to be added to the apex domain. If subdomain is set, it will be appended to the apex domain in  `jx-requirements-eks.yml` file | `string` | `""` | no |
```

Reply:
I'll need to make the changes to this first in outputs.tf and variables.tf, since these tables are generated from these files.